packages/nimble-components/src/rich-text/models/markdown-serializer.ts

```diff
@@ -51,7 +51,15 @@ export class RichTextMarkdownSerializer {
         };
         const marks = {
             italic: defaultMarkdownSerializer.marks.em!,
-            bold: defaultMarkdownSerializer.marks.strong!
+            bold: defaultMarkdownSerializer.marks.strong!,
+            // Autolink markdown in CommonMark flavor: https://spec.commonmark.org/0.30/#autolinks
```

Comment:
```suggestion
            // Autolink markdown in CommonMark flavor: https://spec.commonmark.org/0.30/#autolinks
            // Prosemirror model reference: https://github.com/ProseMirror/prosemirror-markdown/blob/c7210d0e55c82bfb0b2f7cba5dffe804575fafb3/src/to_markdown.ts#L3C1-L26C2
```

Reply:
These comments don't explain why we can't just leverage the `defaultMarkdownSerializer.marks.link` and are forking it with our own implementation.

Reply:
I updated the comment with more details